samples/net/mqtt_azure/src/config.h

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
No need to change this for this PR, but it would be convenient to concentrate all the cloud specific samples to `samples/net/cloud/...` subdirectory. Please prepare such a PR at some point for 2.2 release.